develop-docs/sdk/data-model/envelope-items.mdx

```diff
@@ -181,18 +181,101 @@ details.
 
 ### User Feedback
 
-User Feedback was called User Report in the beginning. Therefore the envelope item type is `"user_report"`.
-The item contains a user feedback / user report JSON payload:
+Item type `"feedback"`. This Item contains an [event payload](/sdk/data-model/event-payloads/) encoded in JSON, with an additional `feedback` context object.
+
+Example payload:
 ```json
-{"event_id":"9ec79c33ec9942ab8353589fcb2e04dc","email":"john@me.com","name":"John Me","comments":"It broke."}\n
+{
+  "event_id": "9ec79c33ec9942ab8353589fcb2e04dc",
+  "timestamp": "2011-05-02T17:41:36Z",
+  "platform": "javascript",
+  "level": "error",
+  "contexts": {
+    "feedback": {
+      "contact_email": "john@example.com",
+      "name": "John Smith",
+      "message": "I love session replay!",
+      "url": "https://sentry.io/replays/",
+      "associated_event_id": "32fd1995636d446385016e2747623e11",
+      "replay_id":"82840977e85b4ed3bc27f7b5b25cec15"
+    }
+  }
+}
 ```
 
-**Attributes**
+**Payload Attributes**
+
+We only document attributes for the `contexts.feedback` object, which is **required**
+for this item type. For other attributes, see [Event Payloads](/sdk/data-model/event-payloads/).
+
+`message`
+
+: **String, required.** Comments of the user, describing what happened and/or sharing
+feedback. The max length is **4096 characters**.
+
+`contact_email`
+
+: *String, recommended.* The email of the user.
+
+`name`
+
+: *String, optional.* The name of the user.
+
+`url`
+
+: *String, optional.* The URL of the referring webpage which may be used to search
```

Comment:
"referring webpage" is a bit unclear, could we use some more specific wording here?

Reply:
URL from which the feedback was submitted?
URL of the page the user was on when the feedback was submitted?

Reply:
Changed to "The URL of the webpage the user was on when submitting feedback."

Although I like the original too. I think "referring webpage" hints at HTTP referrer, which I think this field is pretty similar to?